lkft-fastboot.jinja2

```diff
@@ -1,4 +1,20 @@
-{% extends "lkft.jinja2" %}
+{% extends "master.jinja2" %}
+
+{% block metadata %}
+  {{ super() }}
+  git branch: {{KERNEL_BRANCH}}
```

Comment:
since this is the same in lkft-fastboot and lkft-nfs maybe create a new file (lkft-metadata.jinja2?) and include it here?

Reply:
yes make sense.